.gitignore

```diff
@@ -28,3 +28,5 @@ coverage
 Makefile
 !doc/_sphinx/Makefile
 .bak
+packages/flame_svg/example/macos
+packages/flame_svg/example/windows
```

Comment:
These should be in the `flame_svg` .gitignore

Reply:
I moved them to `flame_svg/example` .gitignore